tests/test_mdlp.py

```diff
@@ -1,16 +1,13 @@
-from sklearn.utils.testing import assert_array_equal
-from sklearn.utils.testing import assert_equal
-from sklearn.utils.testing import assert_almost_equal
-from sklearn.utils.testing import assert_array_almost_equal
+import numpy as np
 
```

Comment:
You can remove the newline here.

Overall, I think there is some work to be done about my conventions for importing packages.